examples/SharpBrick.PoweredUp.Examples/ExampleMoveHubInternalTachoMotorControl.cs

```diff
@@ -0,0 +1,31 @@
+using System.Threading.Tasks;
+using SharpBrick.PoweredUp;
+
+namespace Example
+{
+    public class ExampleMoveHubInternalTachoMotorControl : BaseExample
+    {
+        public override async Task ExecuteAsync()
+        {
+            using (var moveHub = Host.FindByType<MoveHub>())
+            {
+                var internalMotor = moveHub.InternalMotor;
+                await internalMotor.StartSpeedAsync(50, 100);
+                await internalMotor.StartSpeedForTimeAsync(2000, 90, 100, SpecialSpeed.Hold, SpeedProfiles.AccelerationProfile | SpeedProfiles.DecelerationProfile);
+
+                await Task.Delay(2000);
+
```

Comment:
Maybe an additional example could be added to access a single build in motor

```c#
                await Task.Delay(2000);

                await internalMotor.StopByBrakeAsync();
                var internalMotorA = moveHub.A.GetDevice<MoveHubInternalMotor>();
                await internalMotorA.StartSpeedAsync(50, 50);
                await Task.Delay(2000);
                await internalMotorA.StopByFloatAsync();
```

Reply:
Or combine it on the sample. That align with my comments in naming the motors in the hub

Reply:
Good idea.  I've moved out the external motor and now the example shows using the combined motor and individual motors in the hub seperately.